From: Jan Kloetzke <jan@kloetzke.net>
Subject: Fix sysroot if stripping is disabled

When disabling CONFIG_BID_STRIP_BINARIES, BID will just use symlinks when
"installing" libraries. This does not work with the sysroot target when a
SOVERSION is set for the library. To fix that, fall back to a copy.

--- a/mk/lib.mk	2025-05-23 07:50:43.989786347 +0200
+++ b/mk/lib.mk	2025-05-23 07:50:49.230019144 +0200
@@ -44,7 +44,9 @@
                                 $(INSTALL) -m 644 $(1) $(2))$(SOLINK)
 INSTALLFILE_LIB_LOCAL   ?= $(if $(call do_strip,$(1)),                      \
                                 $(call copy_stripped_binary,$(1),$(2),644), \
-                                $(LN) -sf $(abspath $(1)) $(2))$(SOLINK)
+                                $(if $(SOVERSION_FULL),                     \
+                                     $(INSTALL) -m 644 $(1) $(2),           \
+                                     $(LN) -sf $(abspath $(1)) $(2)))$(SOLINK)
 
 INSTALLFILE		= $(INSTALLFILE_LIB)
 INSTALLDIR		= $(INSTALLDIR_LIB)
